components/admin_console/billing/billing_subscriptions/limits.tsx

```diff
@@ -113,30 +112,6 @@ const Limits = (): JSX.Element | null => {
                         icon='icon-message-text-outline'
                     />
                 )}
-                {cloudLimits?.boards?.cards && (
```

Comment:
Note that removing the limit from Stripe side is sufficient to not show this limit.